src/azure-cli/azure/cli/command_modules/iot/custom.py

```diff
@@ -1060,9 +1061,9 @@ def iot_central_app_create(
               sku=appSku,
               template=template)
 
-    createResult = client.apps.create_or_update(
-        resource_group_name, app_name, app)
-    return createResult
+    if no_wait:
+        return sdk_no_wait(no_wait, client.apps.create_or_update, resource_group_name, app_name, app)
+    return LongRunningOperation(cmd.cli_ctx)(client.apps.create_or_update(resource_group_name, app_name, app))
```

Comment:
Just return sdk_no_wait(no_wait, client.apps.create_or_update, resource_group_name, app_name, app) for the whole method, no need to return LongRunningOperation(cmd.cli_ctx)(client.apps.create_or_update(resource_group_name, app_name, app))

Reply:
Same as I thought before 😁